test/unittest/unittest_nntrainer_models.cpp

```diff
@@ -707,6 +707,21 @@ INI conv_multi_stride(
   }
 );
 
+INI conv_uneven_strides(
+  "conv_uneven_strides",
+  {
+    nn_base + "learning_rate=0.1 | optimizer=sgd | loss=cross | batch_size=3",
+        I("input") + input_base + "input_shape=2:5:3",
+    I("conv2d_c1") + conv_base +
+            "kernel_size = 3,3 | filters=4 | stride=3,3" + "input_layers=input",
```

Comment:
in this unittest, can you make input size larger (`2:5:3` -> `2:5:5`) to ensure that stride works in both x and y?
also add another unittest with `2:4:4` to ensure that with uneven stride of both `1` and `2` works?

Reply:
I'd prefer to have x, y dimension different because there used to be a bug regarding this matter. Instead, I'll add input size: 2:4:4, stride: 1, 2

Reply:
> Instead, I'll add input size: 2:4:4, stride: 1, 2

Add input 2:4:4 and stride 2,1 as well.

Reply:
Sure

Reply:
I'll update this in different PR.